package.json

```diff
@@ -20,6 +20,7 @@
     "install-hooks": "mookme init --only-hook --skip-types-selection",
     "lint:fix": "turbo run lint:fix",
     "lint": "turbo run lint",
+    "lint:env": "./dev-tools/dotenv-linter",
```

Comment:
Shouldn't this be `./packages/dotenv-linter`?

Reply:
ah yeah this doesn't even need to be here. relic of my first attempt ty